Simplification of the Databricks Integration tutorial (#2373)
astronomer/docs: learn/airflow-databricks.md

The open source [Astro Databricks provider](https://github.com/astronomer/astro-provider-databricks) provides full observability and control from Airflow so you can manage your Workflows from one place, which enables you to orchestrate your Databricks notebooks from Airflow and execute them as Databricks Workflows.

> This tutorial shows how to use the Astro Databricks provider to run two Databricks notebooks as a Databricks Workflow. It also discusses why you would want to [use Databricks with Airflow](#why-use-airflow-with-databricks) and [Alternative ways to run Databricks with Airflow](#alternative-ways-to-run-databricks-with-airflow), if the Astro Databricks Provider doesn't fit your use case.

Reviewer: ```suggestion
This tutorial shows how to use the Astro Databricks provider to run two Databricks notebooks as a Databricks Workflow. If you use Databricks services other than Databricks Workflows, see [Alternative ways to run Databricks with Airflow](#alternative-ways-to-run-databricks-with-airflow).
```

> ## Quickstart

Reviewer: Same comment here as with the dbt tutorial - I don't want to call this "quickstart", but good to integrate this code snippet in with the intro while linking to the quickstart in the docs. https://github.com/astronomer/astro-provider-databricks/blob/main/quickstart/astro-cli.md

## Prerequisites

- The [Astro CLI](https://docs.astronomer.io/astro/cli/overview).
- The [Astro CLI](https://docs.astronomer.io/astro/cli/overview) with an empty Astro project created by `astro dev init`.
- Access to a Databricks workspace. See [Databricks' documentation](https://docs.databricks.com/getting-started/index.html) for instructions. You can use any workspace that has access to the [Databricks Workflows](https://docs.databricks.com/workflows/index.html) feature. You need a user account with permissions to create notebooks and Databricks jobs. You can use any underlying cloud service, and a [14-day free trial](https://www.databricks.com/try-databricks) is available.
- Access to an [object storage supported by the Astro Python SDK](https://astro-sdk-python.readthedocs.io/en/stable/supported_file.html). This tutorial uses an [AWS S3](https://aws.amazon.com/s3/) bucket.
- Access to a [relational database supported by the Astro Python SDK](https://astro-sdk-python.readthedocs.io/en/stable/supported_databases.html). This tutorial uses [PostgreSQL](https://www.postgresql.org/).

## Step 1: Configure your Astro project

> An Astro project contains all of the files you need to run Airflow locally.

Reviewer: Same comment as with dbt - you can leave this in. We still want the context for how to start a project :)

Author: 😅 I might have gotten ahead of myself a little bit when deleting things here.

    ```

2. Define the following environment variable in your `.env` file. This allows you to serialize Astro Databricks provider objects.

    ```text
    AIRFLOW__CORE__ALLOWED_DESERIALIZATION_CLASSES = airflow\.* astro_databricks\.*
    ```

## Step 2: Create Databricks Notebooks

> You can orchestrate any existing Databricks notebooks in a Databricks Workflow using the Astro Databricks provider. If you don't have Databricks notebooks ready follow the steps below to create two notebooks to use in this tutorial.

Reviewer: ```suggestion
You can orchestrate any existing Databricks notebooks in a Databricks Workflow using the Astro Databricks provider. If you don't have Databricks notebooks ready, follow the steps below to create two notebooks to use in this tutorial.
```

    - **Login**: Your Databricks login username (email).
    - **Password**: Your [Databricks personal access token](https://docs.databricks.com/dev-tools/auth.html#databricks-personal-access-tokens).

## Step 4: Create your DAG

1. In your `dags` folder, create a file called `my_simple_databricks_dag.py`.

> 2. Copy and paste the following DAG code into the file. Make sure to replace the `DATABRICKS_LOGIN_EMAIL` variable with your Databricks login email. If you are using your existing Databricks notebooks you also will have to adjust the notebook names and potentially have to add additional tasks defined with the DatabricksNotebookOperator to orchestrate more than two Databricks notebooks in this Databricks Workflow.

Reviewer: ```suggestion
2. Copy and paste the following DAG code into the file. Replace the `DATABRICKS_LOGIN_EMAIL` variable with your Databricks login email. If you already had Databricks notebooks and did not create new ones in Step 2, replace the notebook names in the DatabricksNotebookOperators.
```
